### PR TITLE
Fix removeValue test failures

### PR DESCRIPTION
**Pull request by fpringvaldsen**
This fixes the failures from the tests added in https://github.com/typesafehub/config/pull/295.

### CODE REVIEW

**Comment by fpringvaldsen** [comment]
@havocp Right now all this PR does is change the expected results of rendering the tests to match the actual results. This seemed to match the method of removal you proposed in your comment on https://github.com/typesafehub/config/pull/295. It seems to me like it's working correctly, but if not I can do some further investigation and make further changes.

File: config/src/test/scala/com/typesafe/config/impl/ConfigDocumentTest.scala [hunk 1]
```diff
@@ -227,6 +227,30 @@ class ConfigDocumentTest extends TestUtils {
...
+        val removed = configDoc.removeValue("a.b")
+        assertEquals("a { }, a { }", removed.render())
+    }
+
+    @Test
+    def configDocumentRemoveOverridden {
+        val origText = "a { b: 42 }, a.b = 43, a { b: { c: 44 }, a : 57 }"
+        val configDoc = ConfigDocumentFactory.parseString(origText)
+        val removed = configDoc.removeValue("a.b")
+        assertEquals("a { }, a { a : 57 }", removed.render())
```
Comment (havocp) [inline config/src/test/scala/com/typesafe/config/impl/ConfigDocumentTest.scala]: This double a is the problem I saw. now you have a path a.a that wasn't in the original.

Comment (fpringvaldsen) [inline config/src/test/scala/com/typesafe/config/impl/ConfigDocumentTest.scala]: I believe it is in the original (the second object contained at a is the map `{ b: {c: 44}, a: 57}`)

Comment (havocp) [inline config/src/test/scala/com/typesafe/config/impl/ConfigDocumentTest.scala]: you're right, doh. I think I meant to put another brace there. Sorry for the noise !

**Comment by fpringvaldsen** [comment]
@havocp I've modified the test in question to bring it more in line with what I think it was supposed to test

**Comment by havocp** [comment]
Thanks :-)

**Pull request merged**